sdk/cosmos/azure_data_cosmos/src/handler/retry_handler.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use crate::retry_policies::resource_throttle_retry_policy::ResourceThrottleRetryPolicy;
+use crate::retry_policies::{RetryPolicy, RetryResult};
+use async_trait::async_trait;
+use azure_core::http::{request::Request, RawResponse};
+use typespec_client_core::async_runtime::get_async_runtime;
```

Comment:
Don't refer to "typespec" in crates. In fact, not even sure why you have a dependency on `typespec` or `typespec_client_core` in your crate. Everything is re-exported from `azure_core`, so only use types (same module path) from `azure_core`.

Reply:
Thanks for the suggestion @heaths. I have refactored the code.